design-system/atoms/table-header/adapters/react-table/react-table.adapter.tsx

```diff
@@ -25,7 +30,20 @@ export function TableHeaderReactTableAdapter<H>({ headerGroups, classNames }: Ta
             const direction = getDirection(header.column.getIsSorted());
 
             return (
-              <th key={header.id} className={cn(slots.header(), classNames?.header)}>
+              <th
+                key={header.id}
+                className={cn(slots.header(), classNames?.header)}
+                colSpan={header.colSpan}
+                style={{
+                  width: styleKernelPort.pxToRem(header.getSize()),
+                  minWidth: header.column.columnDef.minSize
+                    ? styleKernelPort.pxToRem(header.column.columnDef.minSize)
+                    : undefined,
+                  maxWidth: header.column.columnDef.maxSize
+                    ? styleKernelPort.pxToRem(header.column.columnDef.maxSize)
+                    : undefined,
+                }}
```

Comment:
this could go on a useMemo to be more clear

Reply:
Je pourrai pas faire de useMemo c'est dans un map, mais je fais une fonction en dehors